### PR TITLE
Portfolio mobile

### PR DESCRIPTION
**Pull request by Munyabelden**
Mobile version portfolio
Replication of a figma template

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+        <link rel="stylesheet" href="style.css">
+        <title>My Portfolio</title>
+    </head>
+
+    <body>
+        <header class="intro">
+
+            <div class="toolbar">
+
+                    <h1 class="toolbar-logo font-family-roboto">Belden Mugauri</h1>
```
Comment (ghost) [inline index.html]: - [ ] Kindly wrap your logo in an `<a>` tag.This is for three reasons: convention, semantics, and user experience(UX). [Check this link](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for more info.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+
+                    <button class="menu-button" id="menu-btn" aria-controls="nav-bar">
+                        <i class="fa-solid fa-bars"><span class="sr-only">Menu</span></i>
+                    </button>
```
Comment (ghost) [inline index.html]: - [ ] Kindly wrap your hamburger button in a  `<nav>` tag. This is important for semantic reasons

File: index.html [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+            </div> 
+
+
+
+            <div class="primary-content">
+                <img class="elipse-3" src="images\Ellipse 3.jpg" alt="">
+                <img class="elipse-4" src="images\Ellipse 4.jpg" alt="">
+                <img class="elipse-11" src="images\Ellipse 11.png" alt="">
+                <img class="elipse-12"src="images\Ellipse 12 (1).jpg" alt="">
+                <img class="elipse-13" src="images\Ellipse 12.jpg" alt="">
```
Comment (ghost) [inline index.html]: - [ ] Kindly note that these images are supposed to be used in `css ` using  background-image property. [Check this link](https://stackoverflow.com/questions/492809/when-to-use-img-vs-css-background-image) for more info.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+                <div>
+                   <div class="primary-information">
+
+                            <h1 class="primary-title font-family-crete">I'm Belden.<br> Glad to see you!</h1>
```
Comment (ghost) [inline index.html]: - [ ] Kindly wrap your headline section in a `<section>` tag. It represents a generic standalone section of the document [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) for more info.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+                            
+                            <p class="description font-family-poppins">
+                                I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.
+                            </p>
+
+                            <span class="social-header font-family-poppins">LET'S CONNECT</span>
+                    </div>
```
Comment (ghost) [inline index.html]: - [ ] Kindly use either a `<button>` or ` <a>` tag for `let's connect` element for semantic reasons

File: index.html [hunk 6]
```diff
@@ -0,0 +1,94 @@
...
+                            <a href="#" target="_blank" rel="noopener noreferrer">
+                                <i class="fa-brands fa-square-github"><span class="sr-only">github</span></i>
+                            </a>
+                            
+                            <a href="#" target="_blank" rel="noopener noreferrer">
+                                <i class="fa-brands fa-twitter"><span class="sr-only">twitter</span></i>
+                            </a>
+
+                            <a href="#" target="_blank" rel="noopener noreferrer">
+                                <i class="fa-brands fa-linkedin-in"><span class="sr-only">linkedin</span></i>
```
Comment (ghost) [inline index.html]: - [ ] Kindly use `<ul>` and `<li>`  to group and list your social links. This important for semantic reasons

**Review by frank1738 — CHANGES_REQUESTED** [review]
Hi @Munyabelden , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!




## Highlights

✅ Correct use of Github workflow
✅ No linter error


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

N/A
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 7]
```diff
@@ -0,0 +1,90 @@
...
+                            </li>
+                
+                            <li class="nav-item font-family-roboto">
+                                <a href="#">About</a>
+                            </li>
+                
+                            <li class="nav-item font-family-roboto"> 
+                                <a href="#">Contact</a> 
+                            </li>
+                   </ul>
```
Comment (bienvenuushindi) [inline index.html:39]: :orange_circle: [OPTIONAL] Make sure to hide this section on the mobile version, as it causes the page to overflow from the normal flow. Another option is to remove it; there is a special activity for toggling the menu.

fix-this

**Review by bienvenuushindi — APPROVED** [review]
Hi @Munyabelden    , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Charlie Brown

## Highlights
- Github flow is used correctly. ✔
- All linters checks are ok. ✔

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**Pull request merged**